### ISSUE TITLE
Incomplete

### ISSUE THREAD

**Issue by MichaelBrooks-Informatyx** [dotnet/docs#26207]
[Enter feedback here]
This page needs to at least mention (and ideally reference the pages for) child namespaces such as Microsoft.CodeAnalysis.CSharp.Scripting . Perhaps I've just failed to spot it, but looking at this page I would not even know of the existence of the scripting API.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 62003371-6d1c-0ebe-d88c-4ded5e1536d0
* Version Independent ID: a6531341-6be7-cd40-d926-156c28b6231f
* Content: [Microsoft.CodeAnalysis.CSharp Namespace](https://docs.microsoft.com/en-us/dotnet/api/microsoft.codeanalysis.csharp?view=roslyn-dotnet-3.11.0)
* Content Source: [dotnet/xml/ns-Microsoft.CodeAnalysis.CSharp.xml](https://github.com/dotnet/roslyn-api-docs/blob/live/dotnet/xml/ns-Microsoft.CodeAnalysis.CSharp.xml)
* Product: **dotnet-roslyn-api**
* Technology: **microsoft.codeanalysis**
* GitHub Login: @dotnet-bot
* Microsoft Alias: **dotnetcontent**

**Comment by BillWagner** [comment]
Adding @gewarren @mimisasouvanh 

I don't think any of the API pages for namespaces include child namespaces. Should we add this as a feature request?

**Comment by MichaelBrooks-Informatyx** [comment]
Definitely, please Bill - we need some means of knowing of the existence of such.

Regards,
Michael
